src/connector/Einstore.ts

```diff
@@ -423,16 +430,15 @@ export class Einstore {
 		return this.networking.put(`/keys/${id}`, data)
 	}
 
-	public updateUser = async (id: string, data: any): Promise<Response> => {
-		const promise = this.networking.put(`/users/${id}`, data)
-		const res = await promise
-		const json = await res.json()
-		return json
-	}
-
-	public createApiKey = async (team: string, name: string, type: number): Promise<Response> => {
+	public createApiKey = async (
+		team: string,
+		name: string,
+		tags: String,
+		type: number
+	): Promise<Response> => {
 		const promise = this.networking.postJson(`/teams/${team}/keys`, {
 			name: name,
+			tags: tags,
 			type: type,
 		})
```

Comment:
In javascript, this can be written shorter.

```typescript
// These two do the same thing
{
  name: name,
  tags: tags,
  type: type,
}

// if you dont write the part after `:` it will automatically use a variable with the same name.
{
  name,
  tags,
  type,
}
```